src/envs/behavior.py

```diff
@@ -55,14 +53,13 @@ def __init__(self) -> None:
         if not _BEHAVIOR_IMPORTED:
             raise ModuleNotFoundError("Behavior is not installed.")
         self._config_file = modify_config_file(
-            os.path.join(igibson.root_path,
-                         CFG.behavior_config_file), CFG.behavior_task_name,
-            CFG.behavior_scene_name, CFG.behavior_randomize_init_state)
+            os.path.join(igibson.root_path, CFG.behavior_config_file),
+            CFG.behavior_task_name, CFG.behavior_scene_name,
+            False)  # behavior_randomize_init_state = False in config file
```

Comment:
is the comment meant to say that this setting will _always_ be false in the config file?